File: C# Basics/FirstStepsInCoding/CharityCampaign.cs
using System;

namespace CharityCampaign
{
    class Program
    {
        static void Main(string[] args)
        {
            int campaignDays = int.Parse(Console.ReadLine());
            int workers = int.Parse(Console.ReadLine());
            int cakes = int.Parse(Console.ReadLine());
            int waffles = int.Parse(Console.ReadLine());
            int pancakes = int.Parse(Console.ReadLine());

            double cakesPrice = cakes * 45;
            double wafflelPrice = waffles * 5.80;
            double pancakesPrice = pancakes * 3.20;

            double netProfit = (cakesPrice + wafflelPrice+ pancakesPrice) * workers * campaignDays;
            double endProfit = netProfit - (netProfit / 8);

            Console.WriteLine(endProfit);
        }
    }
}

File: Java Basics/FirstStepsInCodingEx/CharityCampaign.java
package FirstStepsInCodingEx;

import java.util.Scanner;

public class CharityCampaign {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        short campaignDays = Short.parseShort(scanner.nextLine());
        short workers = Short.parseShort(scanner.nextLine());
        short cakes = Short.parseShort(scanner.nextLine());
        short waffles = Short.parseShort(scanner.nextLine());
        short pancakes = Short.parseShort(scanner.nextLine());

        double cakesPrice = cakes * 45;
        double wafflesPrice = waffles * 5.80;
        double pancakesPrice = pancakes * 3.20;

        double totalProfit = (cakesPrice + wafflesPrice + pancakesPrice) * campaignDays * workers;
        double raisedFunds = totalProfit - (totalProfit / 8);

        System.out.printf("%.2f", raisedFunds);
    }
}
